specifications/providers/openstack/flavors_and_images.md

```diff
@@ -126,6 +126,10 @@ As defined by this specification we expect:
 ```json
 {
   "properties": {
+    "unikorn:os_name": "Ubuntu",
```

Comment:
So on reflection I think we want....

* kernel: linux/windows (support all images ever)
* familiy: rehdat/debian/arch (defines the package management, e.g. does service x use DEB or RPM)
* distro: redhat/centos/rocky/debian/ubuntu/arch (see below...)
* variant: desktop/server (optional, defines a reasonable expectation of what packages are there by default)
* codename: feisty/gusty (optional)
* version: 24.04 etc.

Why the change of heart?  On reflection, doing the UI stuff, calling `name.toLower().startsWith("ubuntu")` felt just way too hacky.  Now we can just directly match based on the distro and use with https://icon-sets.iconify.design/logos/.  Where a logo doesn't exist we can fallback to the family.  One of the reasons why I like doing stuff with the UI as it exposes these deficiencies!